### PR TITLE
Fixed Django 1.9 compatibility: Pass views as callables, not as strings

### LINKED ISSUE #145: Django 1.9 not support url data
This error occurs when I add: url(r'^tinymce/', include('tinymce.urls')) to urls.py (urlpatterns).
```
urlpatterns = [
    url(r'^admin/', admin.site.urls),
    url(r'^$', views.home, name='home'),
    url(r'^tinymce/', include('tinymce.urls')),
]
```
```
/usr/local/lib/python3.4/site-packages/tinymce/urls.py:10: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got textareas_js). Pass the callable instead.
  url(r'^js/textareas/(?P<name>.+)/$', 'textareas_js', name='tinymce-js'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:11: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got textareas_js). Pass the callable instead.
  url(r'^js/textareas/(?P<name>.+)/(?P<lang>.*)$', 'textareas_js', name='tinymce-js-lang'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:12: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got spell_check). Pass the callable instead.
  url(r'^spellchecker/$', 'spell_check'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:13: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got flatpages_link_list). Pass the callable instead.
  url(r'^flatpages_link_list/$', 'flatpages_link_list'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:14: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got compressor). Pass the callable instead.
  url(r'^compressor/$', 'compressor', name='tinymce-compressor'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:15: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got filebrowser). Pass the callable instead.
  url(r'^filebrowser/$', 'filebrowser', name='tinymce-filebrowser'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:16: RemovedInDjango110Warning: Support for string view arguments to url() is deprecated and will be removed in Django 1.10 (got preview). Pass the callable instead.
  url(r'^preview/(?P<name>.+)/$', 'preview', name='tinymce-preview'),

/usr/local/lib/python3.4/site-packages/tinymce/urls.py:16: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^preview/(?P<name>.+)/$', 'preview', name='tinymce-preview'),
```

### PR DESCRIPTION
**Pull request by hugodes**
Conflicts:
	tinymce/urls.py
Fixes #145

### CODE REVIEW

File: tinymce/urls.py [hunk 1]
```diff
@@ -1,17 +1,18 @@
 # Copyright (c) 2008 Joost Cassee
 # Licensed under the terms of the MIT License (see LICENSE.txt)
+from tinymce.views import textareas_js, spell_check, flatpages_link_list, compressor, filebrowser, preview
```
Comment (Natim) [inline tinymce/urls.py]: Can we do: ``from tinymce import views`` and then use ``views.textareas_js`` in the url definition?

**Comment by hugodes** [comment]
@Natim took your advice into acount in the last commit

**Comment by Natim** [comment]
Thanks.

**Comment by Natim** [comment]
Released with in django-tinymce 2.1.0

**Comment by hugodes** [comment]
That was my first contibution to a django app :D thanks !

**Pull request merged**